packages/react-instantsearch/examples/query-suggestions-rn/Readme.md

```diff
@@ -0,0 +1,28 @@
+This examples shows how to use `react-instantsearch` and `react-native` to display query 
+suggestions along with instant results.
+
+It was created using `exp`. 
+
+## Clone the example
+
+```
+curl https://codeload.github.com/algolia/react-instantsearch/tar.gz/master | tar -xz --strip=2 react-instantsearch-master/packages/react-instantsearch/examples/query-suggestions-rn
+```
+
+## Start the example
+
+You need an expo account. [Create an account or signup with github](https://expo.io/signup). 
```

Comment:
Maybe we should move for `create-react-native-app`, it's a pain to create an account just for run the App locally? WDYT?